timur/lib/client/jsx/components/model_map/model_report.jsx

```diff
@@ -498,7 +498,7 @@ const ModelReport = ({
   }, [model_name, models]);
 
   useEffect(() => {
-    if (counts[model_name]?.count) {
+    if (counts[model_name]?.count >= 0) {
```

Comment:
Hopefully I'm just being dense, but this doesn't seem like it would fix the issue to me?

Reply:
Should be working in production now, if you check the same cytof model you were looking at before? :crossed_fingers:

Reply:
confirmed!